main/src/cgeo/geocaching/storage/DataStore.java

```diff
@@ -2652,7 +2652,9 @@ public static void clearLogOffline(final String geocode) {
 
         init();
 
-        database.delete(dbTableLogsOffline, "geocode = ?", new String[]{geocode});
+        final String[] geocodeWhereArgs = {geocode};
+        database.delete(dbTableLogsOffline, "geocode = ?", geocodeWhereArgs);
+        database.execSQL(String.format("UPDATE %s SET visiteddate = 0 where geocode = ?", dbTableCaches), geocodeWhereArgs);
```

Comment:
Fixed these. 
There are some more places in this class where SQL keywords are lowercase. Maybe I go through them in a separate PR in one go.